Pycloud Deployer/infra_classes.py

```diff
@@ -0,0 +1,288 @@
+from azure.mgmt.resource import ResourceManagementClient
+from azure.mgmt.network import NetworkManagementClient
+from azure.mgmt.compute import ComputeManagementClient
+from modules import (
+    create_dbserver_parameters, 
+    create_publicip_parameters, 
+    create_nic_parameters, 
+    create_webserver_parameters, 
+    run_command_parameters
+)
+
+class WebServer():
+    def __init__(self, subscription_id, token_credential, deployment_name) -> None:
+        """Initiate the WebServer class properties
```

Comment:
It's great to see docstrings but they could be more informative (purpose of each class/method, parameters, return types, and any exceptions they might raise)